dlx_rest/api/utils.py

```diff
@@ -153,6 +154,18 @@ def get(schema_name):
             }
         elif schema_name == 'jfile':
             data = DlxConfig.jfile_schema
+        elif schema_name == 'api.filelist':
+            data = {
+                'type': 'array',
+                'items': {
+                    'type': 'object',
+                    'properties': {
```

Comment:
Should identifier be in this schema?

Reply:
I'm not sure. If you've gotten to this endpoint, you already know the record associated with it